insights/parsers/cpuinfo.py

```diff
@@ -50,6 +52,8 @@
     >>> cpu_info = shared[CpuInfo]
     >>> cpu_info.cpu_count
     2
+    >>> cpu_info.apicid
+    ["2", "2"]
```

Comment:
The test in the `Examples` should be a reflection of the above `Sample input:`

Reply:
@xiangce - I have fixed the issue as per your suggestion, please review